examples/formatter-cabal-gild.toml

```diff
@@ -0,0 +1,6 @@
+# Example generated by ../examples.sh
+[formatter.cabal-gild]
+command = "cabal-gild"
```

Comment:
How should I fix this [CI error][1]?

Change `command` in this example to `cabal-gild-wrapper` or change the name of the wrapper script to `cabal-gild`?  Both of them seem not correct to me.

[1]: https://buildbot.numtide.com/#/builders/312/builds/367

Reply:
I guess just change it to `cabal-gild-wrapper`. Ideally we'd have a note here about how these tools need a wrapper to work. Note how this is broken with other tools that need similar `for file in ` wrapper:

- https://github.com/numtide/treefmt-nix/blob/020cb423808365fa3f10ff4cb8c0a25df35065a3/examples/formatter-statix.toml
- https://github.com/numtide/treefmt-nix/blob/020cb423808365fa3f10ff4cb8c0a25df35065a3/examples/formatter-buf.toml